packages/react/src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -24,9 +25,15 @@ const Breadcrumb = ({
     [customClassName]: !!customClassName,
   });
 
+  const classNameNav = cx({
+    [customClassNameNav]: !!customClassNameNav,
+  });
+
   return (
-    <nav className={className} {...rest}>
-      {children}
+    <nav className={classNameNav} aria-label="Breadcrumb">
```

Comment:
```suggestion
    <nav className={customClassNameNav} aria-label="Breadcrumb">
```

we can just pass this in directly right?